moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -532,34 +482,6 @@ const ProjectView = () => {
                             </ListItemIcon>
                             <ListItemText primary="Rename" />
                           </MenuItem>
-                          {projectStatus?.current_status !== "on hold" && (
-                            <MenuItem
-                              onClick={() => handleUpdateStatus("on hold")}
-                              className={classes.projectOptionsMenuItem}
-                              selected={false}
-                            >
-                              <ListItemIcon
-                                className={classes.projectOptionsMenuItemIcon}
-                              >
-                                <PauseCircleOutlineOutlinedIcon />
-                              </ListItemIcon>
-                              <ListItemText primary="Place on hold" />
-                            </MenuItem>
-                          )}
-                          {projectStatus?.current_status !== "canceled" && (
-                            <MenuItem
-                              onClick={() => handleUpdateStatus("canceled")}
-                              className={classes.projectOptionsMenuItem}
-                              selected={false}
-                            >
-                              <ListItemIcon
-                                className={classes.projectOptionsMenuItemIcon}
-                              >
-                                <CancelOutlinedIcon />
-                              </ListItemIcon>
-                              <ListItemText primary="Cancel" />
-                            </MenuItem>
-                          )}
```

Comment:
If the user wants to cancel or place a project on hold, they must now add the corresponding phase and set it to current.